packages/lib-user/src/components/shared/MainContent/helpers/getDateRangeSelectOptions.spec.js

```diff
@@ -154,7 +154,7 @@ describe('components > MainContent > getDateRangeSelectOptions', function () {
   describe('with a custom date range', function () {
 
     beforeEach(function () {
-      clock = sinon.useFakeTimers(new Date('2023-04-15T00:00:00'))
+      clock = sinon.useFakeTimers(new Date('2023-04-15T00:00:00Z'))
```

Comment:
Previously, the new Date is created in the time zone of the machine running the test. The expectations are UTC, so this failed if the machine running the tests was not set to UTC too.

Reply:
I think the test code blocks starting at line 8 and 81 check that local time is being converted to UTC for the related UI components appropriately, but I'll check the user/group stats pages from a few other time zones again to confirm.

Reply:
I think the tests are ok, just wanted to double check that I'm not accidentally passing something that should fail, by running the clock in a time zone that just happens to work.

Looking at it again, I think it is expected that the outputs would change by one day (and fail), for a `+02:00` time zone, because the tests then are running one day behind the clock in UTC.